doc/howtos/firewall-configuration.rst

```diff
@@ -119,3 +119,11 @@ Opening holes to firewall using this method is not absolutely mandatory.
 If you are building a device without dynamic services or applications,
 you can simply add the necessary rules to the custom ruleset that is
 loaded upon device startup.
+
+Note that if you are starting your service as non-root user (as you
+probably should be) using systemd's ``User=`` option, you'll need to
+make sure that the firewall manipulation commands are run with original
+permissions. An easy way to do that is to use
+``PermissionsStartOnly=true`` option in the service file. This works if
+you don't need to run any other ``ExecStartPre=`` or similar commands
+with the reduced non-root privileges.
```

Comment:
Having that all done by the AppFW generator is definitely great! Can we reflect the current state of affair in this paragraph? It could be as simple as stating that nothing is to be done if using the AppFW generator and a manifest. If that's the case, we should also enhance the AppFW documentation which only mentions nspawn as the container type that is supported and I don't see how to define the ports that the app needs in the manifest.